components/exercises/chapter-4-oscar/DeepComparison.test.tsx

```diff
@@ -0,0 +1,59 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import DeepComparison from './DeepComparison';
+
+// Mock the Utils module
+jest.mock('./Utils', () => ({
+  checkEqual: jest.fn(),
+  lastWeekList: [],
+  currentWeekList: [],
+}));
+
+describe('DeepComparison', () => {
+  test('displays "It is not the same list" when the lists are not equal', () => {
+    render(<DeepComparison />);
+
+    // Click the add button to make the lists not equal
+    fireEvent.click(screen.getByText('+'));
+
+    // Assert that the "It is not the same list" message is displayed
+    expect(screen.getByText('It is not the same list')).toBeInTheDocument();
+  });
+
+  beforeEach(() => {
```

Comment:
This it is usually put before any tests, below the `describe`

Reply:
understood I placed it after describe